spread.yaml

```diff
@@ -1447,4 +1467,37 @@ suites:
                 systemctl status snapd.socket || true
             fi
 
+    tests/kernels/:
+        summary: Test snapd with different kernels
+        backends: [ google-kernels ]
+        systems:
+          - ubuntu-2*
+        manual: true
+        environment:
+          USE_KERNEL/latest: ''
+          USE_KERNEL/version: '$(HOST: echo "$SPREAD_USE_KERNEL")'
+        prepare: |
+            "$TESTSLIB"/prepare-restore.sh --prepare-suite
+        prepare-each: |
+            wget https://raw.githubusercontent.com/pimlie/ubuntu-mainline-kernel.sh/master/ubuntu-mainline-kernel.sh
```

Comment:
Hmm the mainline kernel does not carry some of our confinement patches so we should be mindful of what the result will be. Unless that was exacrtly your intent I would rather use one of the versions of the Ubuntu "sauce" kernels which at least should behave consistently and are legitimately used by a number of our users and customers that are not on HWE kernels.